test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -177,30 +172,17 @@ observations:
           longName: "Altitude of Satellite"
           units: "m"
 
-        - name: "MetaData/startChannel"
-          source: variables/startChannel
-          longName: "Start Channel"
-
-        - name: "MetaData/endChannel"
-          source: variables/endChannel
-          longName: "End Channel"
-
-        - name: "MetaData/channelScaleFactor"
-          source: variables/channelScaleFactor
-          longName: "Channel Scale Factor"
-
         - name: "MetaData/sensorChannelNumber"
           source: variables/sensorChannelNumber
           longName: "Sensor Channel Number"
 
-        # ObsValue
-        - name: "ObsValue/scaledSpectralRadiance"
-          source: variables/scaledSpectralRadiance
-          longName: "Scaled IASI Spectral Radiance"
-          units: "W m-2 sr-1 m"
-
         - name: "MetaData/fractionOfClearPixelsInFov"
           source: variables/fractionOfClearPixelsInFov
           longName: "Fraction of Clear Pixels in a Field of View"
           units: "1"
           range: [0, 1]
+
+        - name: "ObsValue/spectralRadiance"
+          source: variables/spectralRadiance
+          longName: "IASI Spectral Radiance"
+          units: "W m-2 sr-1 m"
```

Comment:
Should the second m be there? I think spectral radiance "W m-2 sr-1 s", and I say "s" because the full unit is "W m-2 sr-1 Hz-1" but since "Hz" is already "s-1", "s-1-1" is just "s".

Reply:
@emilyhcliu

Reply:
This is the definition of scaled/unscaled radiance from the BUFR table.  
```
| SCRA     | 014046 | SCALED IASI RADIANCE               |
| SCRA     |    0             0 |  16 | W M**-2 SR**-1 M |
```

This is spectral radiance.  **W m-2 sr-1 m**  = **W / (m2 sr m-1)**
Radiant flux per unit area (m2) per unit solid angle (sr) per wavenumber (m-1)

Reply:
Ok, that works for me. Thanks for double checking.